sdk/spring/ci.yml

```diff
@@ -55,7 +55,6 @@ trigger:
       - sdk/spring/spring-cloud-azure-stream-binder-eventhubs/pom.xml
       - sdk/spring/spring-cloud-azure-stream-binder-servicebus-core/pom.xml
       - sdk/spring/spring-cloud-azure-stream-binder-servicebus/pom.xml
-      - sdk/spring/spring-cloud-azure-test-eventhubs-binder/pom.xml
```

Comment:
The spring-cloud-azure-test-eventhubs-binder is added back to the 4.0 branch. https://github.com/Azure/azure-sdk-for-java/tree/feature/azure-spring-cloud-4.0/sdk/spring